core/src/main/resources/filter.properties

```diff
@@ -0,0 +1,4 @@
+keywords.1=TransactionImple
+keywords.2=Periodic Recovery
+keywords.3=TransactionSynchronizationRegistryImple
```

Comment:
Didn't Amos suggest a comma separated list of keywords here. Why did you go back to this approach.